subprojects/core/src/main/java/org/gradle/internal/service/scopes/BuildScopeServices.java

```diff
@@ -171,19 +171,12 @@
  * Contains the singleton services for a single build invocation.
  */
 public class BuildScopeServices extends DefaultServiceRegistry {
-    public static BuildScopeServices forSession(BuildSessionScopeServices sessionServices) {
-        return new BuildScopeServices(sessionServices);
-    }
-
-    protected BuildScopeServices(ServiceRegistry parent, StartParameter startParameter) {
-        this(new BuildSessionScopeServices(parent, startParameter, ClassPath.EMPTY));
-    }
-
-    private BuildScopeServices(final BuildSessionScopeServices sessionServices) {
-        super(sessionServices);
+    public BuildScopeServices(final ServiceRegistry parent) {
+        super(parent);
+        addProvider(new BuildScanConfigServices());
```

Comment:
Was this required as part of supporting continuous build? It looks out of place here.